public/app/features/plugins/admin/api.ts

```diff
@@ -120,8 +120,8 @@ async function getLocalPluginReadme(id: string): Promise<string> {
 
 async function getLocalPluginChangelog(id: string): Promise<string> {
   try {
-    const markdown: string = await getBackendSrv().get(`${API_ROOT}/${id}/markdown/CHANGELOG`);
-    const markdownAsHtml = markdown ? renderMarkdown(markdown) : '';
+    const pluginFromGcom = await getBackendSrv().get(`api/gnet/plugins/${id}?version=latest`);
+    const markdownAsHtml = pluginFromGcom ? renderMarkdown(pluginFromGcom.changelog) : '';
```

Comment:
Hmm, what is going to happen to plugins that are not published to GCOM? (Until now we were displaying the README that was available locally, but I guess after this change they won't show any README?)

Currently what we are doing is that [we are fetching the remote metadata from GCOM](https://github.com/grafana/grafana/blob/8044cb50f17a021a32e7ac0b83cf8d723457a9ae/public/app/features/plugins/admin/api.ts#L18), [then fetching the locally available one](https://github.com/grafana/grafana/blob/8044cb50f17a021a32e7ac0b83cf8d723457a9ae/public/app/features/plugins/admin/api.ts#L25), and only falling back to the GCOM version of the README if the local version is not available. Maybe we could change this logic instead to use the GCOM version by default?

Reply:
It's also worth remembering that on-prem installations can be locked down and not have access to the internet.

Reply:
You are both right. I have deleted this peace from this method. And as @leventebalogh showed we already have this logic, we just used it in not correct way. Changing the order of changelog helped.  `changelog: remote?.changelog || localChangelog,` if for some reason we can not get remote plugin we will show local changelog